tests/TestCase/Console/CommandRunnerTest.php

```diff
@@ -311,27 +311,12 @@ public function testRunValidCommandInflection()
      */
     public function testRunValidCommandWithAbort()
     {
-        $app = $this->makeAppWithCommands(['failure' => SampleShell::class]);
+        $app = $this->makeAppWithCommands(['failure' => AbortCommand::class]);
         $output = new ConsoleOutput();
 
         $runner = new CommandRunner($app, 'cake');
-        $result = $runner->run(['cake', 'failure', 'with_abort'], $this->getMockIo($output));
-        $this->assertSame(Shell::CODE_ERROR, $result);
-    }
-
-    /**
-     * Test returning a non-zero value
-     *
-     * @return void
-     */
-    public function testRunValidCommandReturnInteger()
```

Comment:
Is this possible with commands?

Reply:
There's already another test which with integer return value.